src/sqlfluff/core/default_config.cfg

```diff
@@ -114,6 +118,33 @@ spacing_within = inline
 spacing_before = any
 spacing_after = any
 
+# Setting a selection of clauses to "alone" ensures line breaks
+# are preferred around them. Setting this to "alone:strict" would always
```

Comment:
Does "preferred" mean the linter is allowed to add line breaks there? Does it mean anything else/additional?

Reply:
I'll clarify this. What it actually means is that if there's a newline on _one_ side then it will introduce one on the other too - but if there isn't a newline _on either side_ then it won't force new line breaks on either without the `strict` option.

Reply:
This is corrected to:

```
# By setting a selection of clauses to "alone", we hint to the reflow
# algorithm that in the case of a long single line statement, the
# first place to add newlines would be around these clauses.
# Setting this to "alone:strict" would always _force_ line breaks
# around them even if the line isn't too long.
```